### PR TITLE
feat: adds ADC and SBC (a lot of crazy stuff) to enable snake to work

### PR DESCRIPTION
**Pull request by lucascebertin**

### CODE REVIEW

File: src/opcodes.c [hunk 1]
```diff
@@ -55,6 +55,25 @@ void writeOnMemoryOrAccumulator(unsigned int address, unsigned char result, stru
...
+  cpu->status.negative = (cpu->accumulator >> 7);
+  cpu->status.zero = cpu->accumulator == 0;
+}
+
+void adcOpcode(unsigned int address, struct CPU *cpu, struct BUS *bus) {
+  signed int valueFromAddress = readBus(address, bus);
+  sumOperation(valueFromAddress, cpu, bus);
+}
+
+void sbcOpcode(unsigned int address, struct CPU *cpu, struct BUS *bus) {
```
Comment (rafaeldelboni) [inline src/opcodes.c]: could you put this function in alphabetic order?

File: src/opcodes.h [hunk 2]
```diff
@@ -1,6 +1,8 @@
 #include "cpu.h"
 
 int hexToDecimalMode(unsigned char hex);
+void adcOpcode(unsigned int address, struct CPU *cpu, struct BUS *bus);
+void sbcOpcode(unsigned int address, struct CPU *cpu, struct BUS *bus);
```
Comment (rafaeldelboni) [inline src/opcodes.h]: could you put this function in alphabetic order as well?

**Review by rafaeldelboni — APPROVED** [review]

File: src/opcodes.c (hunk 1 above)
Comment (lucascebertin) [reply src/opcodes.c]: Done!

File: src/opcodes.h (hunk 2 above)
Comment (lucascebertin) [reply src/opcodes.h]: Done!

**Pull request merged**